src/server/components/dashboard/listsItems/getViewModel.ts

```diff
@@ -14,7 +15,7 @@ interface DetailsViewModel {
 /**
  * TODO: implement i18n
  */
-const fieldTitles: { [prop: string]: string } = {
+export const fieldTitles: { [prop: string]: string } = {
```

Comment:
Minor point: does this function need to be exported?  Can't see any new usages of it.